astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -2518,6 +2518,10 @@ def igetattr(
                 elif isinstance(inferred, objects.Property):
                     function = inferred.function
                     if not class_context:
+                        if not context.callcontext:
```

Comment:
We may have the identical problem just below this, but I'm hesitant to just add it without a test case in hand. One thing at a time :-)